### PR TITLE
Add Content Security Policy Header Tag

### PR DESCRIPTION
**Pull request by Shriram-Balaji**
Fixes the Content Security Policy Warning in Excalidraw Desktop - https://github.com/excalidraw/excalidraw-desktop/issues/82

### CODE REVIEW

**Comment by Shriram-Balaji** [comment]
Looks like the CI Lint and tests are failing, but GH Action doesn't show why :thinking: . Running the lint and tests locally seem to work though.

**Comment by lipis** [comment]
That's strange...

File: public/index.html [hunk 1]
```diff
@@ -62,7 +62,10 @@
     />
     <!-- OG tags require absolute url for images -->
     <meta name="twitter:image" content="https://excalidraw.com/og-image.png" />
-
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="block-all-mixed-content; child-src 'none'; connect-src http: ws:; default-src 'self'; font-src 'self' data: https: filesystem:; img-src 'self' data: https:; script-src 'self' 'unsafe-inline' https://www.googletagmanager.com https://www.google-analytics.com; style-src 'self' 'unsafe-inline' https:;"
```
Comment (lipis) [inline public/index.html:67]: I'm no CSP expert.. but isn't having just `http:` and `ws:` is very loose?

Comment (Shriram-Balaji) [reply public/index.html]: I Agree, it should be https: and wss:, as we shouldn't ideally allow fetch, XHR and websocket connections over unencrypted connections.

**Pull request merged**